code/game/objects/items/loadout_beacons.dm

```diff
@@ -2418,6 +2439,12 @@ GLOBAL_LIST_EMPTY(loadout_boxes)
 	entry_class = LOADOUT_CAT_PISTOL
 	spawn_thing = /obj/item/storage/box/gun/pistol/cyberpunks
 
+/datum/loadout_box/crimsondeath
```

Comment:
it's not a roundstart loadout gun, its a rare drop loot gun and sorta talked it through in dev channel and they even suggested i buff the damage from 60 to 75 since its insanely slow ROF

Reply:
that one though fenny said its okay i make a gun for myself like that, i just also made a normie version to add to spawnlists so its not a waste